battetl/constants.py

```diff
@@ -1,6 +1,10 @@
 class Constants:
     DEFAULT_TIME_ZONE = 'America/Los_Angeles'
 
+    MAX_RETRIES = 10
```

Comment:
Nitpick, but perhaps this name could be more descriptive. Max retries of what? Where?

Of course, if I had my way, all variable names would be 80 characters long. :P

Reply:
good idea! Added database prefix